google-cloud-clients/google-cloud-bigtable-admin/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-bigtable-admin</artifactId>
+  <version>0.55.2-alpha-SNAPSHOT</version><!-- {x-version-update:google-cloud-bigtable:current} -->
```

Comment:
Update the version-update string to the right artifact

Reply:
done